The SCARD_W_INSERTED_CARD define was simply removed with no substitute 
from the pcsc-lite defines with the update to 1.6.0. Remove it.

Index: common/phones/pcsc.c
===================================================================
--- common/phones/pcsc.c.orig
+++ common/phones/pcsc.c
@@ -199,7 +199,9 @@ static gn_error get_gn_error(PCSC_IOSTRU
 	case SCARD_W_UNPOWERED_CARD:
 	case SCARD_W_RESET_CARD:
 	case SCARD_W_REMOVED_CARD:
+/* removed from pcsc-lite
 	case SCARD_W_INSERTED_CARD:
+*/
 	 	return GN_ERR_SIMPROBLEM;
 	case SCARD_E_NO_MEMORY:
 		return GN_ERR_MEMORYFULL;
